manage.py

```diff
@@ -13,6 +15,25 @@
 manager = Manager(app)
 
 
+def excute_sql_folder(files):
```

Comment:
Thanks, I am sure you have figured out by now that spelling is not my strength. :)